CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 - Started a changelog.
 - Updated pull request template
 
+### ♻️ Refactoring
+
+- Refactored graph and course related route functionality into controllers (#1410)
```

Comment:
Omit the pull request number (we do this for MarkUs but I'm not planning on doing this for the other SDS projects)